clouseau/bugzilla.py

```diff
@@ -4,6 +4,7 @@
 
 import six
 import re
+from urllib.parse import urlencode
```

Comment:
For now we're trying to support both Python 2 and Python 3, could you do something like https://github.com/calixteman/clouseau/blob/master/clouseau/socorro.py#L8 for this import?